app/views/place/_place.html.erb

```diff
@@ -43,7 +43,7 @@
               <% end %>
 
               <% if place["phone"].present? %>
-                <p class="govuk-body">Phone: <%= place["phone"] %></p>
```

Comment:
I assumed the main partial should also be modified as it also doesn't match the standard. Please, let me know if that assumption is incorrect.